File: fabric-frontend/src/app/api/models/query-dto.ts
/* tslint:disable */
/* eslint-disable */
import { NavItem } from './nav-item';
import { PagingDto } from './paging-dto';
export interface QueryDto {
  attributes?: { [key: string]: string };
  environmentKey?: string;
  navItem?: NavItem;
  paging?: PagingDto;
  systemComponentKey?: string;
}

File: fabric-backend-core/src/main/java/io/storydoc/fabric/query/app/QueryDTO.java
package io.storydoc.fabric.query.app;

import io.storydoc.fabric.console.app.navigation.NavItem;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Map;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class QueryDTO {

    private String environmentKey;
    private String systemComponentKey;
    private Map<String, String> attributes;
    private PagingDTO paging;
    private NavItem navItem;
}
